spec/index.html

```diff
@@ -733,125 +733,140 @@ <h3>Literals</h3>
+        <a>language-tagged strings</a> and <a>directional language-tagged strings</a> without
+        the <a>datatype IRI</a> because it is always either
+        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code> (<code>rdf:langString</code>)
+        or <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#dirLangString</code> (<code>rdf:dirLangString</code>), respectively.
+      </p>
+
+      <p>
+        Any [=string=] complying with [[!BCP47]] MAY be used to represent a [=language tag=] in concrete syntaxes or implementations.
+        Such strings MAY be case normalized
+        (for example, by canonicalizing as defined by
+        <a data-cite="bcp47#section-4.5">BCP 47 section 4.5</a>).
+        Alternatively, an implementation MAY preserve the case from the original representation,
+        provided that it processes it in a case-insensitive manner.
+      </p>
+
+      <aside class=note>
+        The treatment of language tags has changed between RDF 1.1 and RDF 1.2.
+        In RDF 1.1, `"chat"@fr` and `"chat"@FR` theoretically represent two distinct terms, but implementations may replace one with the other via some form of normalization.
+        In RDF 1.2, they represent the exact same literal, i.e., the case difference in the concrete syntax does not propagate into the abstract syntax.
+        Since many RDF 1.1 implementations do normalize language tags internally, they will not be impacted by this change.
+      </aside>
+
+    </section>
+
+    <section>
+      <h2>Literal value</h2>
+
+        <p>The <dfn>literal value</dfn> associated with a <a>literal</a> is defined as follows.</p>
+
+      <ul>
+        <li>If the literal is a <a>language-tagged string</a>,
+          then the literal value is a pair consisting of its <a>lexical form</a>
+          and its <a>language tag</a>, in that order.</li>
+        <li>If the literal is a <a>directional language-tagged string</a>, then the literal value is
+          a tuple of its <a>lexical form</a>, its <a>language tag</a>, and its <a>base direction</a>,
+          likewise in that order.</li>
+        <li>If the literal's <a>datatype</a> is handled by an RDF implementation,
+          <ul>
+            <li>if the literal's <a>lexical form</a> is in the <a>lexical space</a>
+              of the <a>datatype</a>, then the literal value is the result of applying
+              the <a>lexical-to-value mapping</a> of the datatype to the
+              <a>lexical form</a>.</li>
+            <li>otherwise, the literal is <dfn data-lt-no-plural>ill-typed</dfn> and no literal value can be
+               associated with the literal. Such a case produces a semantic
+               inconsistency but is not <em>syntactically</em> ill-formed.
+               Implementations SHOULD accept [=ill-typed=] literals and produce RDF
+               graphs from them. Implementations MAY produce warnings when
+               encountering [=ill-typed=] literals.</li>
```

Comment:
```suggestion
            <li>If the literal's <a>lexical form</a> is in the <a>lexical space</a>
              of the <a>datatype</a>, then the literal value is the result of applying
              the <a>lexical-to-value mapping</a> of the datatype to the
              <a>lexical form</a>.</li>
            <li>Otherwise, the literal is <dfn data-lt-no-plural>ill-typed</dfn> and no literal value can be
               associated with the literal. Such a case produces a semantic
               inconsistency, but it is not <em>syntactically</em> ill-formed.
               Implementations SHOULD accept [=ill-typed=] literals and produce RDF
               graphs from them. Implementations MAY produce warnings when
               encountering [=ill-typed=] literals.</li>
```